### PR TITLE
introduce a foreman-devel module

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
requires:
- [ ] https://github.com/theforeman/theforeman-rel-eng/pull/288

**Comment by ekohl** [comment]
> (OTOH, nothing uses this `buildrequires` entry anyway)

Because of this I would do this. On a technical level it doesn't have implications, but avoids duplication.

**Comment by ekohl** [comment]
And now I see you added `foreman` as a dependency to `foreman-devel`, so that makes me reconsider. I'd avoid that potential circular dependency.

File: modulemd/modulemd-foreman-el8.yaml [hunk 1]
```diff
@@ -26,3 +26,24 @@ data:
         platform: ['el8']
         ruby: ['2.7']
         postgresql: ['12']
+...
+---
+document: modulemd
+version: 2
+data:
+  name: foreman-devel
```
Comment (evgeni) [inline modulemd/modulemd-foreman-el8.yaml:34]: I picked `foreman-devel` over `foreman-build` as it's useful for non-build things too, but the name really does not matter much.

**Review by ekohl — APPROVED** [review]

**Pull request merged**